features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunInterface.h

```diff
@@ -19,6 +19,22 @@
 
 #include "MeshInterfaceNanostack.h"
 
+/**
+ * \brief Struct router_information Wi-SUN router dynamic information.
```

Comment:
I think the router_information naming is misleading , since router can be used to refer to nodes. It should be mesh_information or similar. @mikter what do you think?

Reply:
Agree perhaps something needs to say Wi-SUN as this supports other than Wi-SUN meshes

Reply:
This could be just rpl_info as all the information is RPL related

Reply:
Can we go with **ws_rpl_info** ?